.github/workflows/rubocop.yml

```diff
@@ -1,7 +1,13 @@
 name: Rubocop
 
-# Run this workflow every time a new commit pushed to your repository
-on: push
+# Run every time a commit is pushed to master, or a pull request opened against
+# master.
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
I think this can be expanded to run against literally _everything_ by changing it to:

```yaml
on: [ push, pull_request ]
```

I just copied the values from Sorcery to get it running the test suite on my PR.

Reply:
Updated this to run against everything to reflect the previous setting.